generators/statistics.js

```diff
@@ -0,0 +1,121 @@
+const uuid = require('uuid/v1');
```

Comment:
use `uuid/v4`

Reply:
Julien wants to avoid collision as much as possible. Is it the right one for that ?

Reply:
I'm guessing the higher version, the better

Reply:
yes v4 is better